mojira/mojira-discord-bot
# Add a verification system

## Purpose
Add a verification system! Implements #98.
## Approach
To verify their account, a member will need to:
1. Send `!jira verify` (MojiraBot will send them a 15 character token)
2. Comment on a specific Mojira ticket with their token
3. Send a private message to the bot with `link <jira username>`

The bot will check the ticket for their comment, and if the username and code matches, the accounts will be linked.

This also adds:
- A command `!jira who` or `!jira whois` to check the account linked to either a Mojira or a Discord account.
- A method to remove the verification role from a user temporarily, by reacting to the log message.
- A method to remove the verification link from a user, by reacting to the log message with ✂.

## Future work
